src/client/components/ErrorModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { WarningTriangle } from './icons/WarningTriangle'
+import { Modal, ModalProps } from './ModalNew'
+
+type Props = ModalProps & {
+  children: React.ReactNode
+}
+
+const Container = styled.div`
+  padding: 3rem;
+  padding-left: 5rem;
+  position: relative;
+
+  svg {
+    position: absolute;
+    left: 2.5rem;
+    top: 3.25rem;
+  }
+`
+
+export const ErrorHeading = styled.h2`
+  color: ${colorsV3.gray900};
+  margin: 0;
+  margin-bottom: 0.5rem;
+  line-height: 2rem;
+`
+
+export const ErrorText = styled.div`
+  color: ${colorsV3.gray700};
+  margin: 0;
+  line-height: 1.5rem;
+
+  p {
+    margin: 0;
+  }
```

Comment:
Is this needed because we sometimes render markdown here? Just thinking if this component needs to be concerned with that.

Reply:
Yes, it's because of the markdown. I was a bit unsure how to do it another way, do you have any suggestions? :D

Reply:
I would just move this style to the component that also renders the `<Markdown>` component. Just to co-locate things. Since this component doesn't have anything to do with markdown.